cmd/builder/main.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+	"log"
+	"time"
+	"fmt"
+	"flag"
+)
+
+var codeListID = flag.String("code-list-id", "e44de4c4-d39e-4e2f-942b-3ca10584d078", "")
+var instanceID = flag.String("instance-id", "12345", "")
+var dimensionName = flag.String("dimension-name", "Aggregate", "")
+
+func main() {
+
+	driver := bolt.NewDriver()
+
+	connection, err := driver.OpenNeo("bolt://localhost:7687")
+	if err != nil {
+		return
+	}
+	defer connection.Close()
+
+	createInstanceHierarcyConstraints(connection)
+	if err != nil {
+		log.Println(err.Error())
+		return
```

Comment:
Is it worth doing `os.exit(1)` for all/most error events in the main function?

Reply:
Done